src/extension.ts

```diff
@@ -352,3 +358,22 @@ function activateComposeLanguageClient(ctx: vscode.ExtensionContext): void {
 }
 
 //#endregion Language services
+
+const hasSbomContextKey = 'vscode-docker:hasSbom';
+async function checkDockerVersionForSbom(): Promise<void> {
+
+    let hasSbom:boolean = false;
+    const info = await ext.runWithDefaults(client =>
+        client.info({})
+    );
+
+    const dockerPlugins: DockerPlugins[] = info.clientInfo?.Plugins;
+    if (!dockerPlugins || dockerPlugins.length === 0) {
```

Comment:
You can just do:
supportsSbom = toArray(dockerPlugins || []).some((plugin: DockerPlugins) => (plugin.Name || '').toLowerCase() === 'sbom')